### ISSUE TITLE
Error when trying to upload file via Node with uploadFromStream

### ISSUE THREAD

**Issue by lasling** [Workfront/workfront-api#1240]
Unable to upload file via Node.  The example in the repo also does not work.

var data = new NodeFormData__namespace();
                   ^

TypeError: NodeFormData__namespace is not a constructor
    at NodeApi.uploadFromStream (/Users/**/Documents/workfront-api/dist/workfront-api.cjs.js:707:20)
    at /Users/**/Documents/workfront-api/examples/node/upload-an-image.js:39:18
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

**Comment by bhovhannes** [comment]
I guess it has to do with Node version. Which Node version do you use?

**Comment by lasling** [comment]
v19.2.0

I changed the NodeFormData import to 

`import 'form-data'`

and then removed "filename" from 

`data.append('uploadedFile', stream);`

and I have it working locally.  For some reason it doesn't like when you pass the filename into the data.append method.

I'm pretty new to this library, so obviously this isn't a long term fix, but it does seem to get me past my roadblock.

**Comment by bhovhannes** [comment]
My guess is that is related to rollup build. Produced code has:

```javascript
var NodeFormData = require('form-data');

function _interopNamespaceDefault(e) {
    var n = Object.create(null);
    if (e) {
        Object.keys(e).forEach(function (k) {
            if (k !== 'default') {
                var d = Object.getOwnPropertyDescriptor(e, k);
                Object.defineProperty(n, k, d.get ? d : {
                    enumerable: true,
                    get: function () { return e[k]; }
                });
            }
        });
    }
    n.default = e;
    return Object.freeze(n);
}

var NodeFormData__namespace = /*#__PURE__*/_interopNamespaceDefault(NodeFormData);
```

and later, in NodeApi constructor, does:
```javascript
const data = new NodeFormData__namespace();
```

which is where the error happens.

We didn't published a new version of this package for already a long time and there are some dependencies here which can be replaced with modern alternatives. Supporting ESM in Node is one of things which are feasible to do nowadays.

As Workfront has been acquired by Adobe we're in process of standardizing our API and making it available at https://developer.adobe.com. In that vein maintaining this package has low priority. We ourselves mostly use plain fetch api now instead of using `workfront-api` package.

We can look at this issue, but I cannot say when that will happen. If you'll submit a PR we can merge it quickly.

**Comment by bhovhannes** [comment]
Should be fixed in `8.1.1`
